projects/1.16/assets/projectex-forge/projectex/lang/zh_cn.json

```
@@ -0,0 +1,117 @@
+    "block.projectex.collector.tooltip": "对伺服器的TPS很友善。每秒生产EMC一次。",
+    "block.projectex.compressed_refined_link": "压缩精致EMC接口",
+    "block.projectex.compressed_refined_link.tooltip": "与精致EMC接口相似，但它有54个输出槽。",
+    "block.projectex.cyan_collector": "青色能量收集器 [MK9]",
+    "block.projectex.cyan_power_flower": "青色力量花盆栽罐 [MK9]",
+    "block.projectex.cyan_relay": "青色反物质继电器 [MK9]",
+    "block.projectex.dark_collector": "深色能量收集器 [MK2]",
+    "block.projectex.dark_power_flower": "深色力量花盆栽罐 [MK2]",
+    "block.projectex.dark_relay": "深色反物质继电器 [MK2]",
+    "block.projectex.energy_link": "基础能量EMC接口",
+    "block.projectex.energy_link.tooltip": "你可以用这个方块通过EMC收集器来为你的转化桌增加EMC。",
+    "block.projectex.fading_collector": "渐消能量收集器 [MK15]",
+    "block.projectex.fading_power_flower": "渐消力量花盆栽罐 [MK15]",
+    "block.projectex.fading_relay": "渐消反物质继电器 [MK15]",
+    "block.projectex.final_collector": "终极能量收集器 [MK16]",
+    "block.projectex.final_power_flower": "终极力量花盆栽罐 [MK16]",
+    "block.projectex.final_relay": "终极反物质继电器 [MK16]",
+    "block.projectex.green_collector": "绿色能量收集器 [MK10]",
+    "block.projectex.green_power_flower": "绿色力量花盆栽罐 [MK10]",
+    "block.projectex.green_relay": "绿色反物质继电器 [MK10]",
+    "block.projectex.lime_collector": "黄绿色能量收集器 [MK11]",
+    "block.projectex.lime_power_flower": "黄绿色力量花盆栽罐 [MK11]",
+    "block.projectex.lime_relay": "黄绿色反物质继电器 [MK11]",
+    "block.projectex.magenta_collector": "品红色能量收集器 [MK4]",
+    "block.projectex.magenta_power_flower": "品红色力量花盆栽罐 [MK4]",
+    "block.projectex.magenta_relay": "品红色反物质继电器 [MK4]",
+    "block.projectex.orange_collector": "橙色能量收集器 [MK13]",
+    "block.projectex.orange_power_flower": "橙色力量花盆栽罐 [MK13]",
+    "block.projectex.orange_relay": "橙色反物质继电器 [MK13]",
+    "block.projectex.personal_link": "个人EMC接口",
+    "block.projectex.personal_link.tooltip": "与基础能量EMC接口相似，但也能允许你放入与取出物品。",
+    "block.projectex.pink_collector": "粉红色能量收集器 [MK5]",
+    "block.projectex.pink_power_flower": "粉红色力量花盆栽罐 [MK5]",
+    "block.projectex.pink_relay": "粉红色反物质继电器 [MK5]",
+    "block.projectex.purple_collector": "紫色能量收集器 [MK6]",
+    "block.projectex.purple_power_flower": "紫色力量花盆栽罐 [MK6]",
+    "block.projectex.purple_relay": "紫色反物质继电器 [MK6]",
+    "block.projectex.red_collector": "红色能量收集器 [MK3]",
+    "block.projectex.red_power_flower": "红色力量花盆栽罐 [MK3]",
+    "block.projectex.red_relay": "红色反物质继电器 [MK3]",
+    "block.projectex.refined_link": "精致EMC接口",
+    "block.projectex.refined_link.tooltip": "与个人EMC接口相似，但它拥有1个输入槽和9个输出槽。用来做一个像精致存储一样的系统。它也从学习放入输入槽的物品。",
+    "block.projectex.relay.max_transfer": "最大EMC传输：%s/s",
+    "block.projectex.relay.relay_bonus": "续电器加成：%s/s",
+    "block.projectex.relay.tooltip": "对伺服器TPS很友善。每秒只传输EMC一次。",
+    "block.projectex.stone_table": "石质转化桌",
+    "block.projectex.stone_table.learn": "学习",
+    "block.projectex.stone_table.unlearn": "忘却",
+    "block.projectex.violet_collector": "紫罗兰能量收集器 [MK7]",
+    "block.projectex.violet_power_flower": "紫罗兰 力量花盆栽罐 [MK7]",
```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
    "block.projectex.violet_power_flower": "紫罗兰力量花盆栽罐 [MK7]",
```